README.md

```diff
@@ -5,8 +5,12 @@ virtual machine, and standard libraries that together enable Toit programs to ru
 
 ## Jaguar: Live reloading for the ESP32
 
-You can use Toit to develop, update, and restart your ESP32 applications in less than two seconds through
-[Jaguar](https://github.com/toitlang/jaguar). Jaguar is a small Toit program that runs on your ESP32.
+If you are developing for the ESP32, we recommend to use [Jaguar](https://github.com/toitlang/jaguar). It
+automatically downloads a Toit SDK. For host development, see below.
+
+With Jaguar you can use Toit to develop, update, and restart your ESP32 applications in less
+than two seconds through a small service that runs on the device.
+
 It uses the capabilities of the Toit virtual machine to let you update and restart your ESP32
 applications written in Toit over WiFi whenever your source files change. Once set up, it is as easy as:
```

Comment:
I think the text is now a little weird. The meaning of "it" changed in the last paragraph and it feels weird to refer back to an "it" from a previous paragraph.

Reply:
simplified it.
done.